app/models/user.rb

```diff
@@ -6,4 +6,5 @@ class User < ApplicationRecord
   include DeviseTokenAuth::Concerns::User
   validates :first_name, :last_name, :locale, presence: true
   has_many :rents, dependent: :destroy
+  has_many :book_suggestions, dependent: :restrict_with_exception
```

Comment:
Are you sure you want this option? A book suggestion could exist without an User, moreover if you destroy an user or an user delete his account with this option you block the destroy if he has a book suggestion.
I think `:nullify` is a better option here and allow you to keep book suggestion even if the user destroy his account.